src/odemis/acq/stitching/test/tiledacq_test.py

```diff
@@ -244,6 +247,19 @@ def test_whole_procedure(self):
         self.assertIsInstance(data[0], model.DataArray)
         self.assertEqual(len(data[0].shape), 2)
 
+    def test_registrar_weaver(self):
+
+        overlap = 0.05  # Little overlap, no registration
+        area = (0, 0, 0.00001, 0.00001)
```

Comment:
This area is smaller than the fov, so it only stitches 1 image. Suggest to change it to:
```
sem_fov = compute_scanner_fov(self.ebeam)
area = (0, 0, sem_fov[0], sem_fov[1])
```